### PR TITLE
Book detail 3 structure

### PR DESCRIPTION
**Pull request by FedeMacias**
## Summary
- Added a border with shadow to the outer container
- Added a green border below the book title
- Fixed the margins to match the requested 

## Screenshot
Book-description-3-structure

## Trello Card
https://trello.com/c/tdcye37k/28-detalle-de-libro-iii-estructura

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    margin-top: 24px;
+  
+    .title{
+      color: $black;
+      font-size: 32px;
+      font-weight: 700;
+      line-height: 38px;
+    }
+    
+    .title-border{
```
Comment (MMarussi) [inline scss/detail.scss:34]: This border can be defined in the `.title` class, did you try it?

File: scss/detail.scss [hunk 2]
```diff
@@ -4,12 +4,16 @@
 
 .cover-text-container {
   display: flex;
+    height: 416px;
```
Comment (MMarussi) [inline scss/detail.scss:7]: Check the indentation in this line

File: scss/detail.scss [hunk 3]
```diff
@@ -18,13 +22,19 @@
     
     .container-book {
       display: flex;
+      flex-direction: row;
+      width: 540px;
+      margin-top: 24px;
+      margin-bottom: 52px;
+      border-width: 4px;
+      border-bottom-style: solid;
+      border-color: $earls-green;
```
Comment (MMarussi) [inline scss/detail.scss:31]: Have you tried with `border-bottom: 4px solid $earls-green` ?

File: scss/detail.scss [hunk 4]
```diff
@@ -18,13 +22,19 @@
     
     .container-book {
       display: flex;
+      flex-direction: row;
```
Comment (MMarussi) [inline scss/detail.scss:25]: You can remove this rule since it's the default of flex

**Review by MMarussi — CHANGES_REQUESTED** [review]

**Review by MMarussi — APPROVED** [review]

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**